tests/fixtures.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import uuid
+
+from labware_domain_models import BarcodedSbsLabware
+from labware_domain_models import LabwareDefinition
+
+GENERIC_UUID = uuid.uuid4()
```

Comment:
it would probably be better to hardcode this as a specific UUID rather than generating a new one on the fly every time the test is run